lib/perl/Genome/Sys/Lock.pm

```diff
@@ -151,30 +159,63 @@ sub with_default_lock_resource_args {
 
     $args{block_sleep} = 60 unless defined $args{block_sleep};
     $args{max_try} = 7200 unless defined $args{max_try};
+    $args{scope} = $ENV{GENOME_SYS_LOCK_DEFAULT_SCOPE} unless defined $args{scope};
```

Comment:
We can't put `GENOME_SYS_LOCK_DEFAULT_SCOPE` into environment until LIMS is on a snapshot with it so I think we also have to have a hard-coded default?  Plus, anyone that doesn't start a new shell won't have `GENOME_SYS_LOCK_DEFAULT_SCOPE`.